### PR TITLE
DEVCVL-272 Marketpalce UI-UX Improvements

### PR DESCRIPTION
**Pull request by kyawthetkt**
1. DEVCVL-272 UI/UX improvements 
2. Add protected component and create/edit collection, create/edit asset and create auction wrapped in it.

### CODE REVIEW

**Review by pracheebapate — APPROVED** [review]

File: src/components/product/bid/Bid.js [hunk 1]
```diff
@@ -105,6 +105,9 @@ const Bid = ({
             <React.Fragment>
                 <Row justify="center">
                     <Countdown
+                        onFinish={() => {
```
Comment (jibrail-idris) [inline src/components/product/bid/Bid.js:108]: Why `reload()`? Wouldn't the app lose its state?

Comment (kyawthetkt) [reply src/components/product/bid/Bid.js:108]: @jibrail-idris I'd like to get fresh state from api endpoint so I have used `reload()` and react router dom `navigate `method cannot manage it excpet `reload()`.

File: src/App.js [hunk 2]
```diff
@@ -95,23 +114,43 @@ const App = () => (
                             <Route path="/explore" element={<ProductPage />} />
                             <Route
```
[the remark this answers is not in the document]
Comment (kyawthetkt) [reply src/App.js]: Fixed

[the remark this answers is not in the document]
Comment (jibrail-idris) [reply src/App.js]: Thanks.. look cleaner now.

File: src/components/product/bid/Bid.js (hunk 1 above)
Comment (jibrail-idris) [reply src/components/product/bid/Bid.js:108]: how about if you store the default state in a global object or something then restore it to the context?

**Review by jibrail-idris — APPROVED** [review]

**Review by shakil-adv — APPROVED** [review]

**Pull request merged**